pkg/scheduler/framework/plugins/selectorspread/selector_spread.go

```diff
@@ -86,17 +86,17 @@ func (pl *SelectorSpread) Score(ctx context.Context, state *framework.CycleState
 
 	c, err := state.Read(preScoreStateKey)
 	if err != nil {
-		return 0, framework.NewStatus(framework.Error, fmt.Sprintf("Error reading %q from cycleState: %v", preScoreStateKey, err))
+		return 0, framework.AsStatus(fmt.Errorf("Error reading %q from cycleState: %w", preScoreStateKey, err))
 	}
 
 	s, ok := c.(*preScoreState)
 	if !ok {
-		return 0, framework.NewStatus(framework.Error, fmt.Sprintf("%+v convert to tainttoleration.preScoreState error", c))
+		return 0, framework.AsStatus(fmt.Errorf("%+v convert to tainttoleration.preScoreState error", c))
```

Comment:
we shouldn't reach this line, but "%+v" is very unreadable. Just remove it.

"cannot convert saved state to tainttoleration.preScoreState"

Reply:
Done